[2025-07-08T04:05:59.389+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-07-08T04:05:59.405+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: minio_ga4_new_file_detector.branch_if_new_data scheduled__2025-07-07T00:00:00+00:00 [queued]>
[2025-07-08T04:05:59.408+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: minio_ga4_new_file_detector.branch_if_new_data scheduled__2025-07-07T00:00:00+00:00 [queued]>
[2025-07-08T04:05:59.409+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 2
[2025-07-08T04:05:59.414+0000] {taskinstance.py:2327} INFO - Executing <Task(BranchPythonOperator): branch_if_new_data> on 2025-07-07 00:00:00+00:00
[2025-07-08T04:05:59.417+0000] {standard_task_runner.py:63} INFO - Started process 206 to run task
[2025-07-08T04:05:59.419+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'minio_ga4_new_file_detector', 'branch_if_new_data', 'scheduled__2025-07-07T00:00:00+00:00', '--job-id', '171', '--raw', '--subdir', 'DAGS_FOLDER/dag_automatic.py', '--cfg-path', '/tmp/tmpfygt1o9o']
[2025-07-08T04:05:59.420+0000] {standard_task_runner.py:91} INFO - Job 171: Subtask branch_if_new_data
[2025-07-08T04:05:59.429+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.11/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-07-08T04:05:59.451+0000] {task_command.py:426} INFO - Running <TaskInstance: minio_ga4_new_file_detector.branch_if_new_data scheduled__2025-07-07T00:00:00+00:00 [running]> on host 4b2ec626a3e3
[2025-07-08T04:05:59.500+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='minio_ga4_new_file_detector' AIRFLOW_CTX_TASK_ID='branch_if_new_data' AIRFLOW_CTX_EXECUTION_DATE='2025-07-07T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-07-07T00:00:00+00:00'
[2025-07-08T04:05:59.501+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-07-08T04:05:59.507+0000] {baseoperator.py:399} WARNING - BranchPythonOperator.execute cannot be called outside TaskInstance!
[2025-07-08T04:05:59.513+0000] {python.py:237} INFO - Done. Returned value was: spark_write_data
[2025-07-08T04:05:59.513+0000] {branch.py:36} INFO - Branch into spark_write_data
[2025-07-08T04:05:59.513+0000] {skipmixin.py:178} INFO - Following branch spark_write_data
[2025-07-08T04:05:59.526+0000] {skipmixin.py:246} INFO - Skipping tasks [('no_new_data_message', -1)]
[2025-07-08T04:05:59.541+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-07-08T04:05:59.554+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=minio_ga4_new_file_detector, task_id=branch_if_new_data, execution_date=20250707T000000, start_date=20250708T040559, end_date=20250708T040559
[2025-07-08T04:05:59.591+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-07-08T04:05:59.614+0000] {taskinstance.py:3482} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-07-08T04:05:59.615+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
